lib/net.c

```diff
@@ -1214,63 +1269,32 @@ gatekeeper_init_network(struct net_config *net_conf)
 	rte_convert_rss_key((uint32_t *)&default_rss_key,
 		(uint32_t *)rss_key_be, RTE_DIM(default_rss_key));
 
-	/* Initialize pktmbuf pool on each numa node. */
```

Comment:
Bug: I can't comment on the line `if (randomize_rss_key() < 0) {` above, but that block cannot return `-1`, it has to `goto numa;`.